OriginateTheme/Sources/Categories/UIColor+OTHex.m

```diff
@@ -1,16 +1,18 @@
 //
-//  UIImage+OriginateThemeHex.m
-//  OriginateUI
+//  UIImage+OTHex.m
+//  OriginateTheme
 //
 //  Created by Philip Kluz on 2016-05-06.
-//  Copyright (c) 2016 Originate. All rights reserved.
+//  Copyright © 2016 Originate Inc. All rights reserved.
 //
 
-#import "UIColor+OriginateThemeHex.h"
+#import "UIColor+OTHex.h"
 
-@implementation UIColor (OriginateThemeHex)
+@implementation UIColor (OTHex)
 
-+ (instancetype)oth_colorWithHexString:(NSString *)hexString
+#pragma mark - UIColor (OTHex)
+
++ (instancetype)ot_colorWithHexString:(NSString *)hexString
 {
     if ([hexString length] != 6) {
```

Comment:
If you could, please extend this to also allow for RGBA (8 chars wherein the last two are alpha value)